ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_creation.py

```diff
@@ -360,3 +360,29 @@ def test_paddle_empty_like(
         x=x[0],
         dtype=dtype[0],
     )
+
+
+# assign
+@handle_frontend_test(
+    fn_tree="paddle.assign",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")
+    ),
+    test_with_out=st.just(True),
+)
+def test_paddle_assign(
+    dtype_and_x,
+    test_flags,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        x=x[0],
```

Comment:
You are not generating the `output` array. By default it is specific to be `None` which would cause the function to fail. You'll have to generate an `output` array (as mentioned in the official docs) and pass it here as well

Reply:
I have made the change, please check if it's correctly done.